miasm2/arch/arm/sem.py

```diff
@@ -1594,15 +1599,16 @@ def add_block(self, block, gen_pc_updt=False):
 
         it_hints = None
         it_cond = None
-        label = None
+        label = block.label
+        assignments = []
```

Comment:
This var has been added for apparently no reason

Reply:
not a bug